test/fixtures/term_parameters.ftl

```diff
@@ -0,0 +1,8 @@
+-term = { $arg ->
+   *[key] Value
+}
+
+key01 = { -term }
+key02 = { -term() }
+key03 = { -term(arg: 1) }
+key04 = { -term("positional", narg1: 1, narg2: 2) }
```

Comment:
Are positional arguments intended to be legal?

Reply:
They're not used by the runtime to resolve the term so even if they're passed, they're just ignored. We could forbid them in `abstract` for now, in case we find a use for them in the future. How does that sound to you?

Reply:
I wondered about `abstact`, too, but no idea if there's going to be a use or not. I'm not a friend of "for now", either, so, well. Yeah, I filed a documentation bug. No strong opinion.

Reply:
Let's leave them as valid, then. Thanks for filing the issue about the docs.